tests/angular-localForage.js

```diff
@@ -465,4 +477,94 @@ describe('Module: LocalForageModule', function() {
       }, done);
     });
   });
+
```

Comment:
All of this added code is indented 4 spaces, but the rest of the code base is 2-space indent. Please conform, to keep a consistent style.